news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+In osu!catch, the players in the top 50 hardly change over the years, because more and more people go inactive while there is a small inflow of new papers to form a new generation. This is saddening, but if this continues further, I don't think I see a good future for osu!catch T_T
+Fortunately there's actually been more and more players from the other game modes trying osu!catch recently and I'm looking forward to this leading to a good outlook and future hopefully.
+
+I genuinely think osu!catch is really attractive and has discriminated its style and gameplay out of the most rhythm games. 
+So I do recommend you guys to try this game mode out and hopefully get into it.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Who or what was (or still is) your main inspiration to become better at this game mode?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+[ExGon](https://osu.ppy.sh/users/214187) was for sure my main inspiration for a long time (and still is).
+And my current one would be [ArMa79](https://osu.ppy.sh/users/4982799) and [Dahcreeper](https://osu.ppy.sh/users/6926006). They are definitely the ones who helped me become better and let me go further by competing with me and pushing my limits.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Have you ever sustained an injury of a finger, wrist or another part of your hand during your journey? If so, how did you manage to deal with it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Looking back on the last 10 years, luckily not once have I had to take a long break or even quit playing because of serious injuries. But I sometimes suffer from mild pain in my fingers or wrists after a really long session. All I do is just take a small break for a day or even a few days.
```

Comment:
```suggestion
Looking back on the last 10 years, luckily not once have i had to take a long break or even quit playing because of serious injuries. But I sometimes suffer from mild pain in my fingers or wrists after a really long session. All I do is just take a small break for a day or even a few days.
```

Reply:
?????